src/drivers/uavcan/uavcan_main.cpp

```diff
@@ -837,6 +850,11 @@ UavcanNode::Run()
 		uavcan::protocol::param::GetSet::Request req;
 		req.index = _param_index;
 
+		//PX4_DEBUG("send param get set listing");
+
+		// sleep a bit to reduce spike on CAN bus traffic
+		usleep(100);
```

Comment:
Same here

Reply:
will remove this for now. Next CAN drone test we will see if a throttle here is really needed. If it is, I will figure out a better way to do that.

Reply:
great thanks :)